sympy/geometry/hyperbola.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import sympy
+import random
+import math
+import decimal
+import numpy as np
+import matplotlib.pyplot as plt
+from sympy import Symbol, symbols, plot_implicit
+from sympy import Rational
```

Comment:
Please import using fully qualified names as this will help us avoid cyclic imports in the long run.